src/lazyfree.c

```diff
@@ -64,7 +64,7 @@ void lazyFreeLuaScripts(void *args[]) {
 void lazyFreeFunctionsCtx(void *args[]) {
     functionsLibCtx *functions_lib_ctx = args[0];
     size_t len = functionsLibCtxFunctionsLen(functions_lib_ctx);
-    functionsLibCtxFree(functions_lib_ctx);
+    functionsLibCtxFree(functions_lib_ctx, 0);
```

Comment:
```suggestion
    freeFunctionsSync(functions_lib_ctx);
```

Btw, all function from function.h should start with "function"? It is the style so far.

If we keep this as `functionsLibCtxFree(functions_lib_ctx)` = sync, we can add a helper in functions.c, maybe call it `functionsLibCtxFreeGeneric(functions_lib_ctx, int async)`? or another name. Anyway, I think this structure has benefit that we have fewer public functions in functions.h.